server/server.js

```diff
@@ -21,35 +22,50 @@ app.use(bodyParser.urlencoded({
 app.use('/static', express.static(path.join(__dirname, '/static')));
 
 
+
+var bobList = [];
+
+
+function nowPlusXMinutes(numMinutes) {
+  return new Date(Date.now + numMinutes*60000);
+}
+
+
 // Main board
+
+// Define routes
 app.get('/', function (req, res) {
   res.sendFile(__dirname + '/templates/board.html');
 });
 
 // Send data to board
-const controller = require('./routes/controllerRoutes');
+const controller = require('./routes/controllerRoutes')(io, db);
 app.get('/controller', controller.GETindex);
 app.post('/controller', controller.POSTbob);
 app.get('/flavors', controller.GETflavors);
 app.get('/tags', controller.GETtags);
 
-// Start socket
-io.on('connection', function(socket, msg){
-  console.log("user connected");
 
-  socket.emit('all_elements', board_elements);
 
+// Start socket
+io.on('connection', function(socket){
+  console.log("board connected");
+
+  // if (msg === 'board') {
+  console.log("sending all active bobs to board");
+  db.Bob.getAllActiveBobs().then(function (bobList) {
+    socket.emit('all_elements', bobList);
+  });
+  // }
 
   socket.on('add_element', function (msg) {
-    bobbles.elements.push(msg);
-    io.emit('add_element', msg);
+    socket.broadcast.emit('add_element', msg);
 
-    console.log('add_text', msg, '\n All elements:', bobbles.elements);
-    // socket.emit('volumes', JSON.stringify(volumes));
-    // io.emit('vizPositions', [position]);
-    // console.log("sent volumes", volumes);
+    db.Bob.saveBob(msg.data, msg.startDate, msg.endDate, msg.flavor, msg.tags);
   });
 });
 
 
 console.log("FORWARDboard running on port 8080");
+console.log("http://localhost:8080, http://localhost:8080/controller");
+console.log();
```

Comment:
The `... port 8080` log is all good but not sure about the second. Def don't need the third